tests/vmi_multus_test.go

```diff
@@ -1235,3 +1199,22 @@ func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName stri
 	}
 	return false
 }
+
+func validateSriovNetworks(virtClient kubecli.KubevirtClient, vmiPod *k8sv1.Pod, networkName, sriovResourceName string) error {
```

Comment:
I think this is only validating if the resource-name for the network matches.
The current name is too broad.

Maybe something like this: `validatePodKubevirtResourceName`.